pdudaemon/listener.py

```diff
@@ -87,6 +90,26 @@ async def process_request(args, config, daemon):
             return False
         args.hostname = config["aliases"][args.alias]["hostname"]
         args.port = config["aliases"][args.alias]["port"]
+    if args.group:
+        if args.hostname or args.port or args.alias:
+            logger.error("Trying to use hostname or alias or port, when also using group")
+            return False
+        aliases_in_group = []
+        for alias in config.get('aliases', {}).keys():
+            if config["aliases"][alias].get('group', False):
+                aliases_in_group.append(alias)
```

Comment:
Shouldn't filter using the value of `args.group`?

As written, this seems to include *all* aliases which have a group, irrespective of the group name.